weave/trace_server/interface/base_object_classes/human_annotation_column.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from weave.trace_server.interface.base_object_classes import base_object_def
+
+
+class HumanAnnotationColumn(base_object_def.BaseObject):
```

Comment:
note: "HumanAnnotationColumn" is a permanent name. It is sort of like a table name, so we have to be sure we like it (: I think there is a subtle world where there are AnnotationColumns that are not filled out by Humans... which might mean we drop the human part?

Reply:
yep, that sounds right, do we then want a field within the class to denote that it came from a human?

Reply:
There currently is `creator` and `wb_user_id` on the feedback table. I think using those are appropriate (which should be automatic right now)